src/api/uploads/BaseUpload.js

```diff
@@ -106,7 +106,7 @@ class BaseUpload extends Base {
 
         const uploadHost = this.getUploadHostFromUrl(upload_url);
         // The default upload host should always be reachable
-        if (uploadHost === `${DEFAULT_HOSTNAME_UPLOAD}/`) {
+        if (uploadHost === `${DEFAULT_HOSTNAME_UPLOAD}/` || uploadHost === `${DEFAULT_HOSTNAME_UPLOAD_APP}/`) {
```

Comment:
can we leave a comment here about why we are matching on the 2 default upload hosts?

Reply:
@satishboggarapu in first line of comment, did you mean to write:
```// DEFAULT_HOSTNAME_UPLOAD is the original default upload host, while DEFAULT_HOSTNAME_UPLOAD_APP is recent.``` ?

Reply:
@manishPh Fixed it.